lighthouse-core/gather/computed/manifest-values.js

```diff
@@ -14,11 +14,7 @@ const PWA_DISPLAY_VALUES = ['minimal-ui', 'fullscreen', 'standalone'];
 // For more discussion, see https://github.com/GoogleChrome/lighthouse/issues/69 and https://developer.chrome.com/apps/manifest/name#short_name
 const SUGGESTED_SHORTNAME_LENGTH = 12;
 
-class ManifestValues extends ComputedArtifact {
-  get name() {
-    return 'ManifestValues';
-  }
-
+class ManifestValues {
```

Comment:
I believe someone once told me that he thought it was silly we made classes with all static members instead of plain objects ;)

FWIW I kinda like them as classes for the flashy `require` name consistency, but I was mostly persuaded back then for plain objects, thoughts?

Reply:
> I believe someone once told me that he thought it was silly we made classes with all static members instead of plain objects ;)

haha, I didn't even think about it as I changed things. There used to be some tsc benefit due to polymorphic `this` (but not an issue with this implementation) and then from augmenting things in javascript passed out on `module.exports` (but that's all fixed in 3.1).

I'll admit to stockholm syndrome in that I don't see it as ugly and unnecessary any more, but I also don't have any problem with a plain object. The main thing might be the `name` property, which I think we still want, whether explicit or from just the `StaticClass.name` default from declaring a class.

Reply:
true true, this WFM :)